app/src/main/java/com/github/codetanzania/ui/activity/ReportIssueActivity.java

```diff
@@ -470,19 +461,11 @@ public void selectLocation(double lats, double longs) {
         showOptionalDetails(null);
     }
 
-    @Override
-    public void onPhotoCaptured(String mCapturePath) {
-        if (mCurrentFragment instanceof ImageCaptureFragment) {
-            mCurrentFragment.getArguments().putString(ImageCaptureFragment.KEY_CAPTURED_IMAGE, mCapturePath);
-        }
-        // we will use it to encode picture data to a 64-bits encoded string
-        mIssueBody.put("pictureFile", mCapturePath);
-    }
 
     @Override
     public void onRemovePreviewItemClicked() {
-        if (mCurrentFragment instanceof IssueDetails2Fragment) {
-            ((IssueDetails2Fragment) mCurrentFragment).removePreviewImageFragment();
+        if (mCurrentFragment instanceof IssueDetailsFormFragment) {
+            ((IssueDetailsFormFragment) mCurrentFragment).removePreviewImageFragment();
             this.optionalBitmapAttachment = null;

```

Comment:
Note: ideally this should be inside the image fragment instead of inside the activity, as it will then be more easily reused.